test-case/check-kmod-load-unload.sh

```diff
@@ -91,8 +91,7 @@ do
         die "Found error(s) in kernel log after module insertion"
 
     dlogi "checking if firmware is loaded successfully"
-    # Check every 1s for 10s
-    if poll_wait_for 1 10 sof_firmware_boot_complete --since=@"$KERNEL_CHECKPOINT"; then
+    if poll_wait_for 1 "$MAX_WAIT_FW_LOADING" sof_firmware_boot_complete --since=@"$KERNEL_CHECKPOINT"; then
```

Comment:
so MAX_WAIT_FW_LOADING always have a  valid data?

Reply:
MAX_WAIT_FW_LOADING has default value of 70 if not overrided by user